src/pages/Facility/locations/LocationNavbar.tsx

```diff
@@ -45,7 +48,7 @@ export function LocationTreeNode({
       pathParams: { facility_id: facilityId },
       queryParams: {
         parent: location.id,
-        mode: "kind",
+        mode: showAllForms ? undefined : "kind",
```

Comment:
```suggestion
           ...(!showAllForms && { mode: "kind" }),
```

to show beds inside a room as well.